.eslintrc.cjs

```diff
@@ -85,6 +85,36 @@ module.exports = {
 
 		// Prevent the use of any in type annotation.
 		'@typescript-eslint/no-explicit-any': 'error',
+
```

Comment:
Looks like 045ad224df2751ff95f010195b62bb93a7fe2925 wasnt redundant, since without it types arent grouped at the bottom 😅